prado/src/main/java/com/schibsted/spain/prado/extensions/Extensions.kt

```diff
@@ -28,6 +29,11 @@ fun Context.buildPradoGalleryIntent(imageUrls: List<String>,
       .putExtra(PradoGalleryActivity.EXTRA_IMAGE_PROVIDER, imageProviderType.name)
 }
 
+@JvmOverloads
```

Comment:
Not really because we are not using this method outside kotlin. It's a copy-paste error... Will remove it. Thanks!